addons/sourcemod/scripting/shavit-misc.sp

```diff
@@ -827,6 +835,17 @@ void UpdateClanTag(int client)
 	{
 		return;
 	}
+	Action result = Plugin_Continue;
+	Call_StartForward(gH_Forwards_OnClanTagChangePre);
```

Comment:
This needs to be moved down to where `gH_Forwards_OnClanTagChange` is currently.